mod/metrics/spec/lib/card/fixed_company_answer_query_spec.rb

```diff
@@ -0,0 +1,372 @@
+require "./test/seed"
+
+RSpec.describe Card::FixedCompanyAnswerQuery do
```

Comment:
these tests were moved from what is now the +answer card specs.  Feels right; they're really testing answer query code. (Also the other tests were getting long and complicated).